docs/tokenomics/nns/nns-intro.md

```diff
@@ -1,218 +1,53 @@
+Neuron owners face challenges in manually voting on every NNS proposal due to high submission volumes and a potential lack of expertise. The NNS allows automatic voting through delegation where neurons automatically vote by following others for all proposals or a specific proposal topic. This ensures efficient decision-making without requiring constant manual input.
 
-* **ManageNetworkEconomics (#NetworkEconomics)**: this is a single proposal type which can update one or several economic parameters:
+It is expected that a large proportion of the overall supply of ICP will be locked to earn rewards. This secures the Internet Computer’s network governance, by making it both difficult and exorbitantly expensive for an attacker to acquire a sufficiently large stake to gain significant influence. Since neuron owners may wish to maximize their rewards by voting on all proposals, most neurons will either be actively managed or configured to follow other neurons so that they can vote automatically.
```

Comment:
I think this is a repetition of above. As already mentioned there I am not sure if we want to mention this.
Do you think this is important? What is the main message we want to deliver here?

Reply:
I think we were aiming to convey this idea:

"Since the incentive to stake ICP in a neuron is high, it is difficult and exorbitantly expensive for an attacker to acquire a sufficiently large stake to gain significant influence."

Does this sound better?